src/components/ui/DbLink.js

```diff
@@ -57,10 +58,14 @@ class TooltipBase extends React.PureComponent {
 			showTooltip,
 			showName,
 			name,
+
+			className,
```

Comment:
Mmh... I'm not a _huge_ fan of passing classes through components. Sort of ties the components together.

The margins can be done in the wrapper, the sizes... not sure. Possibly sanest as a prop though.

Reply:
Personally not a huge fan of introducing a fuck ton of wrapper divs for things, but it does seem more sane